### PR TITLE
S staging stage

### PR DESCRIPTION
**Pull request by the-smooth-operator**

### CODE REVIEW

File: kubernetes/users/discourse-staging-rbac.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: rbac.authorization.k8s.io/v1
```
Comment (LeoMcA) [inline kubernetes/users/discourse-staging-rbac.yaml:1]: Should this file be renamed to `discourse-stage-rbac.yaml`?

File: terraform/stage.tfvars [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#############
```
Comment (LeoMcA) [inline terraform/stage.tfvars:1]: Should we remove the `terraform/staging.tfvars` file, or does that have to happen later?

File: kubernetes/users/discourse-staging-rbac.yaml (hunk 1 above)
Comment (the-smooth-operator) [reply kubernetes/users/discourse-staging-rbac.yaml:1]: yes, good catch!

File: terraform/stage.tfvars (hunk 2 above)
Comment (the-smooth-operator) [reply terraform/stage.tfvars:1]: I want that to happen later. I'd need that file to do `terraform destroy` in that environment, so all the AWS resources will get deleted. After that I'll submit a new PR with the removal of that file

**Review by LeoMcA — APPROVED** [review]
Looks good to me

**Pull request merged**